ffda-usb-wan-hotplug/files/etc/init.d/gluon-usbmuxd

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh /etc/rc.common
+# shellcheck disable=SC2034 # some variables are from /etc/rc.common
```

Comment:
Comment for the reasoning in the line preceding the shellcheck parameter?

Reply:
Not sure I understand:
You want me to move the comment into a separate line?

Reply:
Correct. Move the comment one line aboce the shellcheck parameter.

Reply:
fixed